src/items/external-blocks.md

```diff
@@ -29,9 +29,12 @@
 > _NamedFunctionParametersWithVariadics_ :\
 > &nbsp;&nbsp; ( _NamedFunctionParam_ `,` )<sup>\*</sup> _NamedFunctionParam_ `,` `...`
 
-External blocks form the basis for Rust's foreign function interface.
-Declarations in an external block describe symbols in external, non-Rust
-libraries.
+External blocks provide _declarations_ of items that are not _defined_ in the
```

Comment:
I think this could be more explicit that these are "imports" of a sort.

Reply:
I've tried to clarify that.